.circleci/config.yml

```diff
@@ -14,7 +14,7 @@
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: ${AWS_ECR_DOMAIN}/globality-build:2019.20.1
+    - image: ${AWS_ECR_DOMAIN}/globality-build:faaff99432ff46b774e40a2f6a44e628e9955070
```

Comment:
Pinning this to image generated by globality-corp/globality-build#68 to try to fix [twine issue](https://circleci.com/gh/globality-corp/microcosm/457)